src/Akeneo/Pim/Enrichment/Product/back/Test/Integration/EnrichmentProductTestCase.php

```diff
@@ -182,4 +200,18 @@ protected function createUser(string $username, array $stringRoles, array $group
 
         return $user;
     }
+
+    protected function getUserId(string $username): int
+    {
+        $query = <<<SQL
+            SELECT id FROM oro_user WHERE username = :username
+        SQL;
+        $stmt = $this->get('database_connection')->executeQuery($query, ['username' => $username]);
+        return intval($stmt->fetchColumn());
```

Comment:
```suggestion
        $id = $stmt->fetchOne();
        Assert::notNull($id);

        return \intval($id);
```

To check the user exists (otherwise the current function returns 0)